draft-gpew-priv-ppm.md

```diff
@@ -585,6 +586,12 @@ type 'outdatedConfig'. Clients SHOULD invalidate any cached aggregator
 `HpkeConfig` and retry with a freshly generated Report. If this retried report
 does not succeed, clients MUST abort and discontinue retrying.
 
+The leader MUST ignore any report whose nonce contains a timestamp that falls in
+a batch interval for which it has received at least one collect request from the
+collector. (See {{pa-collect}}.) Otherwise, comparing the aggregate result to
+the previous aggregate result may result in a significant privacy violation.
```

Comment:
nit: this adjective suggests that we have some kind of classification of privacy violations.
```suggestion
the previous aggregate result may result in a privacy violation.
```